forced_photometry/multiband_photometry.md

```diff
@@ -128,7 +126,9 @@ except ImportError:
 coords = SkyCoord('150.01d 2.2d', frame='icrs')  #COSMOS center acording to Simbad
 
 #how large is the search radius, in arcmin
-radius = 48.0 * u.arcmin  
+radius = 0.5 * u.arcmin  #start with a manageable radius and increase as needed
+#full area of COSMOS is radius = 48'
+#running this code on the full COSMOS area takes ~24 hours on a 128core server
```

Comment:
I would avoid inline comments, and in general move comment above the code line. Silly convention, but python styling likes to leave one space after the comment char and two before if it's inline.

```suggestion
# full area of COSMOS is radius = 48'
# running this code on the full COSMOS area takes ~24 hours on a 128core server, therefore
# start with a manageable radius and increase as needed
radius = 0.5 * u.arcmin  

```